### PR TITLE
Add doc for prometheus

### PR DESCRIPTION
**Pull request by lluunn**
/cc @kunmingg 
/cc @jlewi

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/website/233)

### CODE REVIEW

**Comment by jlewi** [comment]
/assign @kunmingg 

Kunming please review this when you can.

File: content/docs/guides/monitoring.md [hunk 1]
```diff
@@ -129,3 +129,38 @@ details on the dashboard.
...
+ks generate prometheus prom --projectId=YOUR_PROJECT --clusterName=YOUR_CLUSTER --zone=ZONE
+ks apply YOUR_ENV -c prom
+```
+
+The prometheus server will scrape the services with annotation `prometheus.io/scrape=true`.
+See [here](https://github.com/kubeflow/kubeflow/blob/master/kubeflow/core/prometheus.yml#L75) for more detail,
+and [here](https://github.com/kubeflow/kubeflow/blob/master/kubeflow/core/metric-collector.libsonnet#L83) for an example.
+
+#### Export metrics to Stackdriver
+The Prometheus server will export metrics to Stackdriver, as configured
```
Comment (abhi-g) [inline content/docs/guides/monitoring.md]: reword ", as configured [here]" to " as [configured]"?

File: content/docs/guides/monitoring.md [hunk 2]
```diff
@@ -129,3 +129,38 @@ details on the dashboard.
...
+Kubeflow provides a Prometheus [component](https://github.com/kubeflow/kubeflow/blob/master/kubeflow/core/prometheus.libsonnet).
+To deploy the Prometheus component:
+
+```
+ks generate prometheus prom --projectId=YOUR_PROJECT --clusterName=YOUR_CLUSTER --zone=ZONE
+ks apply YOUR_ENV -c prom
+```
+
+The prometheus server will scrape the services with annotation `prometheus.io/scrape=true`.
+See [here](https://github.com/kubeflow/kubeflow/blob/master/kubeflow/core/prometheus.yml#L75) for more detail,
```
Comment (abhi-g) [inline content/docs/guides/monitoring.md]: Rewrite this sentence as:
See for more [details](..) and an [example](..).

**Comment by lluunn** [comment]
I think we should keep the Stackdriver part as some GCP users might not want to deploy prometheus.

File: content/docs/guides/monitoring.md [hunk 3]
```diff
@@ -128,4 +128,39 @@ details on the dashboard.
...
+```
+
+The prometheus server will scrape the services with annotation `prometheus.io/scrape=true`.
+See for more [detail](https://github.com/kubeflow/kubeflow/blob/master/kubeflow/core/prometheus.yml#L75) 
+and an [example](https://github.com/kubeflow/kubeflow/blob/master/kubeflow/core/metric-collector.libsonnet#L83).
+
+#### Export metrics to Stackdriver
+The Prometheus server will export metrics to Stackdriver, as
+[configured](https://github.com/kubeflow/kubeflow/blob/master/kubeflow/core/prometheus.yml#L127).
+We are using an [image](https://github.com/kubeflow/kubeflow/blob/master/kubeflow/core/prometheus.libsonnet#L170)
```
Comment (jlewi) [inline content/docs/guides/monitoring.md]: Lunkai can you clarify the instructions here. If I understand the discussion with Kunming the prometheus component saves users the trouble of having to follow the GKE docs. Can we clarify that this is what the prometheus component is doing and if they use it they don't need to follow the instructions in the GKE docs.

**Comment by lluunn** [comment]
https://github.com/kubeflow/kubeflow/issues/1635

PTAL

**Pull request merged**